docs/SPIRVRepresentationInLLVM.rst

```diff
@@ -74,6 +75,24 @@ mangled as __spirv_{TypeName}, where {TypeName} is the name of the SPIR-V
 type with "OpType" removed, e.g., OpTypeEvent is mapped to spirv.Event and
 mangled as __spirv_Event.
 
+Address spaces
+--------------
+
+The following
+`SPIR-V storage classes <https://www.khronos.org/registry/spir-v/specs/unified1/SPIRV.html#Storage_Class>`_
+are naturally represented as LLVM IR address spaces with the following mapping:
+
+SPIR-V storage class    LLVM IR address space
+Function                No address space or addrspace(0)
+CrossWorkgroup          addrspace(1)
+UniformConstant         addrspace(2)
+Workgroup               addrspace(3)
+Generic                 addrspace(4)
+
+SPIR-V extensions are allowed to add new storage classes. For example,
+SPV_INTEL_usm_storage_classes extension adds DeviceOnlyINTEL and HostOnlyINTEL
+storage classes which are mapped to addrspace(5) and addrspace(6) appropriately.
```

Comment:
Do you mean "respectively" instead of "appropriately"?

Reply:
```suggestion
storage classes which are mapped to addrspace(5) and addrspace(6) respectively.
```

Reply:
Thanks.
Fixed.